src/Pages/Auth/Register/styles.js

```diff
@@ -0,0 +1,30 @@
+import { css } from "emotion"
+
+export const emailInput = css`
+  margin-bottom: 24px !important;
+  color: var(--primary-90);
+  font-size: 0.78rem;
+  line-height: 1.44;
+  text-transform: uppercase;
+  font-weight: 700;
+  letter-spacing: 0.75px;
+  input {
```

Comment:
We shouldn't make the input green to indicate success. This was only added by Kostas for demonstration purposes